apps/privacy-panel/js/root/main.js

```diff
@@ -28,6 +28,7 @@ function(panels, SettingsListener, SettingsHelper) {
 
       this.panel = document.getElementById('root');
       this.backBtn = this.panel.querySelector('#back-to-settings');
+      this.fromSettings = null;
```

Comment:
by default, `this.fromSettings` would be `false`. (Make its type consistent would be more understandable)